cmd/bootkube/render.go

```diff
@@ -196,3 +217,28 @@ func altNamesFromURLs(urls []*url.URL) *tlsutil.AltNames {
 	}
 	return &an
 }
+
+func pickDNSServiceIP(serviceCIDR string) (string, error) {
+	ip, ipnet, err := net.ParseCIDR(serviceCIDR)
+	if err != nil {
+		return "", err
+	}
+	// pick the 10th IP as a conventional default
+	for i := 0; i < 10; i++ {
+		incIPv4(ip)
```

Comment:
Seems reasonable. Maybe only print this if it's not the current assumed value `10.3.0.10`?

https://github.com/kubernetes-incubator/bootkube/blob/58536436a4813ca8ceb28692ddba9a308083f954/hack/multi-node/user-data.sample#L32